Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -67,10 +68,7 @@ protected FormsAppCompatActivity()
 			_previousState = AndroidApplicationLifecycleState.Uninitialized;
 			_currentState = AndroidApplicationLifecycleState.Uninitialized;
 			PopupManager.Subscribe(this);
-
-			var anticipator = new Anticipator();
-			anticipator.AnticipateClassConstruction(typeof(Resource.Layout));
-			anticipator.AnticipateClassConstruction(typeof(Resource.Attribute));
+			RuntimeHelpers.RunClassConstructor(typeof(Anticipator).TypeHandle);
```

Comment:
This spins up the thread pool and sets its threads racing ahead of the UIThread to compute and cache results needed during startup.